fullscreen.bs

```diff
@@ -314,6 +314,9 @@ attribute's getter must run these steps:
 <ol>
  <li><p>Let <var>promise</var> be a new promise.
 
+ <li><p>If <var>doc</var> is not <a>fully active</a>, reject <var>promise</var> with a
```

Comment:
then reject* (I realize it's not consistent, but I'd like new and changed text to conform to it at least.)